llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -247,6 +247,24 @@ static OperandInfo getOperandInfo(const MachineInstr &MI,
     llvm_unreachable("Configuration setting instructions do not read or write "
                      "vector registers");
 
+  // Vector Loads and Stores
+  // Vector Unit-Stride Instructions
+  // Vector Strided Instructions
+  /// Dest EEW encoded in the instruction and EMUL=(EEW/SEW)*LMUL
+  case RISCV::VSE8_V:
+  case RISCV::VSM_V:
```

Comment:
VSM should use `OperandInfo(getEMULEqualsEEWDivSEWTimesLMUL(0, MI), 0)`

Reply:
We don't have any mask producing instructions in isSupportedInstr at the moment, so I've decided to remove VSM from this patch.